telegram/bot.py

```diff
@@ -3064,6 +3125,38 @@ def get_chat_members_count(
     ) -> int:
         """Use this method to get the number of members in a chat.
 
+        Args:
+            chat_id (:obj:`int` | :obj:`str`): Unique identifier for the target chat or username
+                of the target supergroup or channel (in the format ``@channelusername``).
+            timeout (:obj:`int` | :obj:`float`, optional): If this value is specified, use it as
+                the read timeout from the server (instead of the one specified during creation of
+                the connection pool).
+            api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
+                Telegram API.
+        Returns:
+            :obj:`int`: Number of members in the chat.
+        Raises:
+            :class:`telegram.error.TelegramError`
+        """
+        warnings.warn(
+            '`bot.get_chat_members_count` is depreciated. '
```

Comment:
```suggestion
            '`bot.get_chat_members_count` is deprecated. '
```
Also apply in `test_bot.py`